demos/basic_simulator/basic_simulator.py

```diff
@@ -0,0 +1,157 @@
+""" implements a very basic simulator, fully actuated and no contact ! """
+
+import numpy as np
+import pinocchio as pin
+from pinocchio.robot_wrapper import RobotWrapper
+import os, sys, time   
+import matplotlib.pyplot as plt 
+
+
+
+
+class CubeSimulator(object):
+    def __init__(self, 
+                urdf_path="/home/vagraval/blmc_ei/workspace/src/catkin/simulators/pybullet_fingers/test/basic_simulator/model/urdf/free_flyer.urdf", 
```

Comment:
Absolute paths should not be hardcoded, otherwise this won't work on other machines.  You can use `rospkg` to dynamically get the path to the pybullet_fingers (see [here](https://github.com/open-dynamic-robot-initiative/blmc_robots/blob/7c5a2c06929aa6d60bb05872fd0507cadd81e8c6/demos/demo_real_finger.py#L83) for an example).